cluster/charts/crossplane/README.md

```diff
@@ -272,30 +112,12 @@ and their default values.
 | `imagePullSecrets`               | Names of image pull secrets to use                              | `dockerhub`                                            |
 | `replicas`                       | The number of replicas to run for the Crossplane operator       | `1`                                                    |
 | `deploymentStrategy`             | The deployment strategy for the Crossplane operator             | `RollingUpdate`                                        |
-| `clusterPackages.aws.deploy`       | Deploy AWS package                                                | `false`
```

Comment:
Do we remove the feature that allows providers to be installed together with Crossplane?

Reply:
We are replacing it with the new packages, tracked: https://github.com/crossplane/crossplane/issues/1726